scenarios/kubernetes_e2e.py

```diff
@@ -537,7 +537,7 @@ def main(args):
 
     if args.use_logexporter:
         # TODO(fejta): Take the below value through a flag instead of env var.
-        runner_args.append('--logexporter-gcs-path=%s' % os.environ.get('GCS_ARTIFACTS_DIR', ''))
+        runner_args.append('--logexporter-gcs-path=%s' % os.environ.get('LOG_DUMP_DIR', os.environ.get('GCS_ARTIFACTS_DIR', '')))
```

Comment:
I would prefer to see the TODO addressed, and the custom path passed in explicitly as a flag
```python
runner_args.append('--logexporter-gcs-path=%s' % args.logexporter_gcs_path)
# ...
parser.add_argument(
        '--logexporter-gcs-path',
        default=os.environ.get('GCS_ARTIFACTS_DIR',''),
        help='GCS path where logexporter tool will upload logs if enabled')
```

Reply:
Thanks! Looks much better now!

PTAL